pkg/kubectl/cmd/version.go

```diff
@@ -50,73 +58,90 @@ func NewCmdVersion(f cmdutil.Factory, out io.Writer) *cobra.Command {
 		Long:    "Print the client and server version information for the current context",
 		Example: versionExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			err := RunVersion(f, out, cmd)
-			cmdutil.CheckErr(err)
+			options := new(VersionConfig)
+			cmdutil.CheckErr(options.Complete(cmd))
+			cmdutil.CheckErr(options.Validate())
+			cmdutil.CheckErr(options.Run(f, out))
 		},
 	}
 	cmd.Flags().BoolP("client", "c", false, "Client version only (no server required).")
 	cmd.Flags().BoolP("short", "", false, "Print just the version number.")
-	cmd.Flags().String("output", "", "output format, options available are yaml and json")
+	cmd.Flags().String("output", "", "one of 'yaml', 'json', or ''")
```

Comment:
Sure, I can remove it.
For what it's worth, `kubectl version --output=` works just fine for me. Maybe you were seeing a network error?